cypress/pages/forgotPassword.page.spec.js

```diff
@@ -0,0 +1,14 @@
+export class ForgotPasswordPage {
+  elementsForgotPasswordPage = {
```

Comment:
```suggestion
  elements = {
```
Навіщо дублювати? Ти все одно будеш використовувати елементи тільки з класом ForgotPasswordPage тож буде зрозуміло куди належать ці елементи

Reply:
Те ж саме і з іншими класами